draft-ietf-httpbis-message-signatures.md

```diff
@@ -289,6 +289,18 @@ NOTE: '\' line wrapping per RFC 8792
 
 Note: these are shown here using the line wrapping algorithm in {{RFC8792}} due to limitations in the document format that strips trailing spaces from diagrams.
 
+Any HTTP field component identifiers MAY have the following parameters in specific circumstances.
+
+sf
```

Comment:
I _think_ you need to allow the top-level type of the field to be specified here (Dictionary, List, or Item) -- that information is necessary input.

Reply:
It was my understanding that this information would be known based on the field itself, by both sides. So putting this on `Representation-Siganture` you know that it's always a Dictionary. I wouldn't want to have a way to try and signal that within the signature parameters.